src/decoder/mod.rs

```diff
@@ -167,9 +170,18 @@ impl<R: Read> Decoder<R> {
         self.limits = limits;
     }
 
+    /// Ignore parse text(text/ztxt/itxt) chunk while read meta data
+    pub fn ignore_text_chunk(&mut self) {
+        self.ignore_text_chunk = true;
+    }
```

Comment:
The style for other options is `fn set_*(&mut self, Value)`. I don't particularly care about the `set` prefix—although the API guidelines recommend reserving pure names without prefix for getters if there are any–but would prefer taking a `bool` parameter here.